smdebug_rulesconfig/profiler_rules/rules.py

```diff
@@ -285,6 +286,8 @@ def __init__(self, **rule_parameters):
         rule_classes = ProfilerReport.get_rules()
         rule_names = str([rule.__name__ for rule in rule_classes]).strip("[]")
         rule_classes_by_name = {rule.__name__.lower(): rule for rule in rule_classes}
+        validate_boolean(self.__class__.__name__, "opt_out_telemetry", opt_out_telemetry)
+        self.opt_out_telemetry = opt_out_telemetry
```

Comment:
Remove this line. We don't want to save parameters of this rule directly as attributes of the class, they should only be saved in the `rule_parameters` dictionary.